.github/workflows/pythonapp.yml

```diff
@@ -25,14 +57,16 @@ jobs:
 
     - name: Install plastimatch
       run: |
-       sudo apt-get install plastimatch
+        sudo apt-get update
+        sudo apt-get install plastimatch
 
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip wheel setuptools
         pip install -r pre-requirements.txt
```

Comment:
shouldn't we be using poetry to do the installations?

Reply:
Poetry has *lot* of problems. I'll create an Issue to discuss.

Reply:
The intention has been to use poetry; however, sorting dependencies has been a blocker.

Reply:
In the Windows + Python 3.10 test:
`platform win32 -- Python 3.10.10, pytest-7.2.2, pluggy-1.0.0 -- C:\hostedtoolcache\windows\Python\3.10.10\x64\python.exe`

is win32 indicating a 32 bit platform or just a legacy name for MS Windows?

Reply:
> The intention has been to use poetry; however, sorting dependencies has been a blocker.

There are some issues on Poetry:
1. It cannot install `pyradiomics` (which is not following pep517)
2. The `*requirements.txt` files are out-of-sync with `pyproject.toml`
3. `poetry.lock` is not tracked in the repo
4. Poetry is slow (slightly)
5. From my experiments, I manually sync the `pyproject.toml` with workable dependencies from `*requirements.txt`, after a workaround to install `pyradiomics`, it fails more test cases than traditional `requirements.txt`

Reply:
And personally, I prefer the repo to have the install, build, test instruction with the source code (markdown) rather than wiki page. Otherwise, the wiki page and the software itself will out-of-sync.
Wiki page is better for a brief instruction, trouble shooting, or release notes, etc.

Reply:
PS.
The previous github action yaml file was using requirements.txt while the wiki page is using poetry (but no poetry.lock file)

Reply:
> In the Windows + Python 3.10 test: `platform win32 -- Python 3.10.10, pytest-7.2.2, pluggy-1.0.0 -- C:\hostedtoolcache\windows\Python\3.10.10\x64\python.exe`
> 
> is win32 indicating a 32 bit platform or just a legacy name for MS Windows?

Legacy name.
It shows win32 on my x64 python+windows

Reply:
Hi @didymo , after this PR, I'll work on Poetry with workarounds and experimentals.

Reply:
G'day @celeron533 , the poetry branch has been updated for your use. The pyproject.toml needs an update which I am attempting to source.